package.json

```diff
@@ -8,7 +8,7 @@
     "test": "test"
   },
   "scripts": {
-    "build": "node script/build.js && npm run compile-sass && npm run autoprefix",
+    "build": "rm -rf lib/{js,css} && npx babel src/js --out-dir lib/js && npm run compile-sass && npm run autoprefix",
```

Comment:
I'm not really sure how this would work (if this belongs in the realm of this project or in a downstream project installing us-forms-system), but sourcemaps are really useful, and seems like there's a way to include them with babel-cli: https://babeljs.io/docs/en/babel-cli.html#compile-with-source-maps.

Reply:
I'll open a new ticket for that, it's a good idea. At minimum we'll needt to add the `--source-maps` flag and commit the resulting `.map` files.